src/administrator/includes/rbacl.php

```diff
@@ -154,10 +154,12 @@ public static function check($userId, $client, $action, $contentId = null)
 	 */
 	public static function authorise($userId, $client, $action, $contentId = null)
 	{
+		$client = (string) $client;
+		$action = (string) $action;
 		$user = Factory::getUser($userId);
 		$result = $user->authorise($action, $client);
 
-		return $result && self::check($userId, $client, $action);
+		return $result && self::check($userId, $client, $action, $contentId);
```

Comment:
@thite-amol you need to look at this.

Reply:
@thite-amol  - Please check this comments